searchguard/rolesmapping.py

```diff
@@ -43,6 +44,18 @@ def check_rolemapping_exists(role):
         raise CheckRoleMappingExistsException('Unknown error checking whether role mapping for {} exists'.format(role))
 
 
+def view_all_rolemappings():
+    """Returns the properties for the requested role mapping if it exists"""
+    view_all_sg_rolemapping = requests.get('{}/rolesmapping/'.format(settings.SEARCHGUARD_API_URL),
+                                           auth=settings.SEARCHGUARD_API_AUTH)
+
+    if view_all_sg_rolemapping.status_code == 200:
+        return json.loads(view_all_sg_rolemapping.text)
+    else:
+        # Could not fetch valid output
+        raise ViewAllRoleMappingException('Unknown error retrieving all role mapping')
```

Comment:
I personally would have done something like this here:
```
if view_all_sg_rolemapping.status_code != 200:
    raise ViewAllRoleMappingException('Unknown error retrieving all role mapping')
return json.loads(view_all_sg_rolemapping.text)
```

Reply:
True, however this is more in line with the style of the rest of the repo. Should I still change it?

Reply:
no just a suggestion